verticapy/tests_new/plotting/highcharts/machine_learning/test_highcharts_elbow.py

```diff
@@ -0,0 +1,92 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+# Pytest
+import pytest
+
+# Standard Python Modules
+
+
+# Other Modules
+
+
+# Verticapy
+from verticapy.learn.model_selection import elbow
+from verticapy.tests_new.plotting.conftest import get_xaxis_label, get_width, get_height
+
+# Testing variables
+COL_NAME_1 = "PetalLengthCm"
+COL_NAME_2 = "PetalWidthCm"
+
+
+class TestHighchartsMachineLearningElbowCurvePlot:
+    """
+    Testing different attributes of Elbow Curve plot
+    """
+
+    @pytest.fixture(scope="class")
+    def plot_result(self, iris_vd):
+        """
+        Create a elbow curve plot
+        """
+        return elbow(input_relation=iris_vd, X=[COL_NAME_1, COL_NAME_2])
+
+    @pytest.fixture(autouse=True)
+    def result(self, plot_result):
+        """
+        Get the plot results
+        """
+        self.result = plot_result
+
+    def test_properties_output_type(self, plotting_library_object):
+        """
+        Test if correct object created
+        """
+        # Arrange
+        # Act
+        # Assert - checking if correct object created
+        assert isinstance(self.result, plotting_library_object), "Wrong object created"
+
+    def test_properties_xaxis_label(self):
+        """
+        Testing x-axis label
+        """
+        # Arrange
+        test_title = "Number of Clusters"
+        # Act
+        # Assert - checking if correct object created
+        assert get_xaxis_label(self.result) == test_title, "X axis label incorrect"
+
+    @pytest.mark.slow
+    @pytest.mark.notcritical
```

Comment:
Interesting marks! I wonder if you're planning to treat the tests with these marks differently.

Reply:
Yes, in case we wanted to increase the efficiency of our tests further, we can setup pytest to skip "notcritical" or "slow" tests.